op-challenger/flags/flags.go

```diff
@@ -105,14 +111,15 @@ var (
 // requiredFlags are checked by [CheckRequired]
 var requiredFlags = []cli.Flag{
 	L1EthRpcFlag,
-	DGFAddressFlag,
+	FactoryAddressFlag,
 	TraceTypeFlag,
 	AgreeWithProposedOutputFlag,
 }
 
 // optionalFlags is a list of unchecked cli flags
 var optionalFlags = []cli.Flag{
 	AlphabetFlag,
+	DGFAddressFlag,
```

Comment:
The `DGFAddressFlag` here should be renamed. The `DGF` stands for `dispute game factory` which doesn't make sense since it is a **single** dispute game.

Reply:
oh wow, that was always a bad name for it. Fixed.